src/app.js

```diff
@@ -1,29 +1,26 @@
-//Importamos configuracion de Express
-import app from './config/express.js';
-
-//Importamos rutas
-import routes from './routes/index.route.js';
-
-import mongoose from 'mongoose';
-
+//Para usar require, hay que importar esto
 import { createRequire } from 'module';
-
 const require = createRequire(import.meta.url);
-
 require('dotenv').config();
 
+//Importamos configuracion de del servidor
+import app from './config/server.js';
+
+// BD Mongo
+import mongoose from 'mongoose';
+
+/** DB connection */
 mongoose
-  .connect('mongodb://mongo/mydatabase', {
+  .connect('mongodb://localhost/data-base', {
```

Comment:
The connection string for the database is one of the things suitable to be included in a config file. When you are testing, you may want to use a different configuration from when you deploy the app. Plus, I think you missed the `port`. 

That reminds me of something pretty important you are not doing, **tests** ⚠️